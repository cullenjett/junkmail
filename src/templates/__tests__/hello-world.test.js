import React from 'react';
import { shallow } from 'enzyme';

import HelloWorld from '../hello-world';

const setup = newProps => {
  const props = {
    ...newProps
  };

  const enzymeWrapper = shallow(<HelloWorld {...props} />);

  return {
    props,
    enzymeWrapper
  };
};

describe('<HelloWorld />', () => {
  it('renders as expected', () => {
    const { enzymeWrapper } = setup();
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
