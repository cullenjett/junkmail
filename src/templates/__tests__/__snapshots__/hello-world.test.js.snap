// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelloWorld /> renders as expected 1`] = `
<Email>
  <h2>
    Hello, world!
  </h2>
  <p>
    Lorem ipsum dolor amet polaroid cloud bread ramps kickstarter kale chips helvetica meggings everyday carry, hot chicken post-ironic master cleanse. Live-edge banjo shoreditch fingerstache wayfarers palo santo post-ironic artisan pop-up brooklyn. Seitan migas neutra gluten-free, franzen cred distillery williamsburg retro jean shorts. Subway tile food truck portland biodiesel cred. Crucifix cornhole fam bushwick.
  </p>
</Email>
`;
